decidim-core/app/helpers/decidim/sanitize_helper.rb

```diff
@@ -16,10 +16,11 @@ def self.included(base)
     #
     # Returns an HTML-safe String.
     def decidim_sanitize(html, options = {})
+      scrubber = options[:scrubber] || Decidim::UserInputScrubber.new
```

Comment:
This change is intentional. Since we added 2 scrubbers in 0.26, we need to allow the new scrubbers being used.